compiler/src/dotty/tools/io/PlainFile.scala

```diff
@@ -78,7 +77,7 @@ class PlainFile(val givenPath: Path) extends AbstractFile {
   }
 
   /** Is this abstract file a directory? */
-  def isDirectory: Boolean = givenPath.isDirectory
+  val isDirectory: Boolean = givenPath.isDirectory
```

Comment:
We should add a comment here
```suggestion
  val isDirectory: Boolean = givenPath.isDirectory // cached for performance on Windows
```